src/pages/EditMonitor.vue

```diff
@@ -317,6 +317,7 @@ export default {
             },
             acceptedStatusCodeOptions: [],
             dnsresolvetypeOptions: [],
+            friendlyNameRequiredOptions: [],
```

Comment:
You can enter options here directly because the array is not populated dynamically

Reply:
I put it with the other two in `init()` as it helps to keep it consistent with them. `dnsresolvetypeOptions` isn't dynamic either so it kind of makes sense to keep them consistent.

Reply:
ok, no problem